### PR TITLE
New package: mpDris2, MPRIS 2 support for mpd

### PR DESCRIPTION
**Pull request by pjones**

### CODE REVIEW

File: pkgs/tools/audio/mpdris2/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  buildInputs = [ autoconf automake intltool ];
+  propagatedBuildInputs = [ python wrapPython ];
+  pythonPath = [ mpd pygtk dbus pynotify ];
+
+  preConfigure = "autoreconf -vfi";
+  postInstall = "wrapPythonPrograms";
+
+  meta = with stdenv.lib; {
+    description = "mpDris2 provides MPRIS 2 support to mpd.";
```
Comment (pSub) [inline pkgs/tools/audio/mpdris2/default.nix]: Descriptions should not end with a period and contain the package name. See http://nixos.org/nixpkgs/manual/#idm140737319009616

File: pkgs/tools/audio/mpdris2/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  src = fetchurl {
+    url = "https://github.com/eonpatapon/${name}/archive/${version}.tar.gz";
+    sha256 = "0zdmamj2ldhr6y3s464w8y2x3yizda784jnlrg3j3myfabssisvz";
+  };
+
+  buildInputs = [ autoconf automake intltool ];
+  propagatedBuildInputs = [ python wrapPython ];
+  pythonPath = [ mpd pygtk dbus pynotify ];
+
+  preConfigure = "autoreconf -vfi";
```
Comment (pSub) [inline pkgs/tools/audio/mpdris2/default.nix]: I am no expert for autoreconf, but just adding `autoreconfHook` to `buildInputs` should suffice.

File: pkgs/tools/audio/mpdris2/default.nix (hunk 1 above)
Comment (pjones) [inline pkgs/tools/audio/mpdris2/default.nix]: Fixed.

File: pkgs/tools/audio/mpdris2/default.nix (hunk 2 above)
Comment (pjones) [inline pkgs/tools/audio/mpdris2/default.nix]: Yep, that works.  Fixed.

**Comment by pSub** [comment]
Thanks!

**Pull request merged**